pkg/status/controller.go

```diff
@@ -196,7 +354,7 @@ func (c *Controller) enqueueObject(obj interface{}) {
 		utilruntime.HandleError(err)
 		return
 	}
-	c.workqueue.Add(ref)
+	c.workqueue.Add(workStatusRef(ref.String()))
```

Comment:
FYI: The `handleObject` and `enqueueObject` methods of Controller are poorly named. I had assumed they apply to workload objects. But actually they apply to WorkStatus objects. It would be better if their names said so.

Not a blocker.